README.md

```diff
@@ -1 +1,33 @@
-# node-hello-world-lab
\ No newline at end of file
+# Node Hello World
+
+## Objectives
+
+1. Implement Hello World in Node
+1. Run the script from the terminal
+
+
+## Introduction
+
+In this lab, you'll practice how to run Node scripts from command line. In addition, you'll develop your first Node application—Hello World, because such app is a must when learning a new language.
+
+## Instructions
+
+1. Create file `hello-world.js` and open it in your favorite editor
+2. Output "Hello World" with `console.log()` which works similarly to browser JavaScript, i.e., prints text.
+3. Launch the script from command line / terminal / command prompt with `$ node hello-world` (`.js` is optional).
```

Comment:
@vicfriedman that's a common notation to symbolize the command prompts. I used it in 11 books that I wrote and never had a person mistaken `$` for the command (although it's possible. I assume students know it already since they are coming from Rails and browser JavaScript)
This should stay.